RELEASE.md

```diff
@@ -16,6 +16,7 @@
 ## Bug fixes and other changes
 * Use default `False` value for rich logging `set_locals`, to make sure credentials and other sensitive data isn't shown in logs.
 * The Kedro IPython extension now surfaces errors when it cannot load a Kedro project.
+* Documented the pipeline registry.
```

Comment:
We usually don't add separate release notes for features that are added in the same release. You could add this to the note for the functionality to make it clear there's docs about how it works.

Reply:
Yeah, makes sense. I added this because the pipeline registry itself was missing docs (not just the autodiscovery component), but I have no strong feelings and can easily remove this. :) Shall I?
```suggestion
```

Reply:
Yes 🙂